pyvrp/Model.py

```diff
@@ -180,6 +180,9 @@ def add_edge(
             """
             warn(msg, ScalingWarning)
 
+        if frm == to and (distance != 0 or duration != 0):
+            raise ValueError("A self-loop edge must have 0 distance/duration.")
```

Comment:
```suggestion
            raise ValueError("A self loop edge must have 0 distance and duration.")
```
Nit: here it's self-loop, in all other places self loop.

Can we add this case to the error condition in the docstring? Something like
> When either distance or duration is a negative value, or when self loops have nonzero distance or duration values.

It'd then also make sense to move this if/raise block up to just below the other error condition, so all error conditions are grouped together.